core/tests/protractor_desktop/embedding.js

```diff
@@ -113,6 +113,11 @@ describe('Embedding', function() {
     '\'X-Frame-Options\' to \'deny\'.',
     'chrome-error:\/\/chromewebdata\/ - Failed to load resource: the server ' +
     'responded with a status of 404 ()',
+    'chrome-error:\/\/chromewebdata\/ - Failed to load resource: the server ' +
+    'responded with a status of 400 ()',
+    'chrome-error:\/\/chromewebdata\/ - Refused to display ' +
+    '\'http:\/\/localhost:9001\/\' in a frame because it set ' +
+    '\'X-Frame-Options\' to \'deny\'.'
```

Comment:
Have you checked that when these errors are raised the page still works correctly? Before ignoring them, we should make sure they don't impact the user experience.

Reply:
I have looked into E2E working and copied the same procedure in localhost and it seems to work correctly.

Reply:
Also in develop I ran the same e2e test & it failed with
First try:  ```Expected $.length = 2 to equal 0.```
Second try:  ```Expected $.length = 1 to equal 0.```

Reply:
> I have looked into E2E working and copied the same procedure in localhost and it seems to work correctly.

@Nik-09 thanks for checking this. When you followed the procedure in localhost, were these console errors raised? We should only be ignoring console errors if we know that they appear when everything works correctly.

> Also in develop I ran the same e2e test & it failed with
> First try: Expected $.length = 2 to equal 0.
> Second try: Expected $.length = 1 to equal 0.

This is helpful, but it doesn't mean that these errors should be ignored. For example, there might be a flake in the tests that raises these console errors. In that case the correct solution is to fix the flake, not to ignore the error. Does that make sense?

Reply:
Yes, thank you it makes sense.
I have checked in the console and it raises the error that I have added not the present errors.
On further investigation -- I have found that ```idToBeReplaced``` is an invalid exploration id, thus initially it was raising 404 error but after schema validation they were checked before reaching to the handler class and concluded as invalid as a result 400 error is raised.

I have removed the current error messages(with 404 status code) and introduced new messages(with 400 status code).

The reason behind failure of ```idToBeReplaced``` in schema validation layer because it fails to match with the regex as provided the handler's schema.

Please correct me if I am wrong at any point.
Thank you.